docs/components/modeler/bpmn/multi-instance/multi-instance.md

```diff
@@ -35,6 +35,8 @@ In case of a **parallel** multi-instance activity, all instances are created whe
 
 A multi-instance activity must have an `inputCollection` expression that defines the collection to iterate over (e.g. `= items`). Usually, it [accesses a variable](/components/concepts/expressions.md#access-variables) of the process instance that holds the collection. The expression is evaluated on activating the multi-instance body. It must result in an `array` of any type (e.g. `["item-1", "item-2"]`).
 
+If you need to iterate `n` times, you can use a for-loop to create an array for a range: `for i in 1..n return i`.
+
```

Comment:
@jonathanlukas what do you think about the following suggestion?
- wrapping the tip into an Admonitions tip box
- linking to the FEEL docs

```suggestion
:::tip

If you need to iterate `n` times (like with a loop-cardinality), you can use the following expression with a [for-loop](/components/modeler/feel/language-guide/feel-control-flow.md#for-loops): `for i in 1..n return i`.

:::
```